dashboard/app/helpers/levels_helper.rb

```diff
@@ -182,6 +182,17 @@ def app_options
       # currently, all levels are Blockly or DSLDefined except for Unplugged
       @app_options = view_options.camelize_keys
     end
+
+    @app_options[:dialog] = {
+      skipSound: !!(@level.properties['options'].try(:[], 'skip_sound')),
+      preTitle: @level.properties['pre_title'],
+      fallbackResponse: @fallback_response.to_json,
+      callback: @callback,
+      app: @level.type.underscore,
```

Comment:
The values were just hardcoded in a bunch of views, but they all follow this format. According to code-studio/src/js/reporting.js, `app` is supposed to be "The app name, as defined by its model." 

`@fallback_response` and `@callback` are defined in `ScriptLevelsController#present_level`.